coreax/kernels/scalar_valued.py

```diff
@@ -162,25 +163,25 @@ def __check_init__(self):
             raise ValueError("'output_scale' must be positive")
 
     @override
-    def compute_elementwise(self, x: ArrayLike, y: ArrayLike) -> Array:
+    def compute_elementwise(self, x, y):
```

Comment:
The override inherits the type hints, and these `ArrayLike` type hints are now incorrect. If I wanted to include the correct ones here I would need to include every single override for every single kernel function which would add 100s of lines unnecessarily

Reply:
Unfortunately, this doesn't work with Sphinx for me. Take a look at the compiled documentation. Does it work for you?

Reply:
Hmm you're right, that is strange because VSCode recognises the override and updates the type hints. What do you suggest we should do? Adding overrides for every single kernel function is obviously possible but it is a lot of bloat.

Reply:
Left for #795.